### ISSUE TITLE
Color::INDIGO is not indigo

### ISSUE THREAD

**Issue by SeanMcLoughlin** [bevyengine/bevy#1193]
**Bevy version**

Bevy 4.0.0

**Operating system & version**

Ubuntu 20.04

**What you did**

Was looking at the [breakout example](https://github.com/bevyengine/bevy/blob/master/examples/game/breakout.rs) where a `ClearColor` resource was added. I noticed the `Color` struct has some constant values to choose from, including the value `INDIGO`. When using that value on a simple default app, the color generated is much more purple than indigo. A simple reproducible example:

```rust
use bevy::prelude::*;

fn main() {
    App::build()
        .add_plugins(DefaultPlugins)
        .add_resource(ClearColor(Color::INDIGO))
        .run();
}
```

**What you expected to happen**

The background color to be indigo.

**What actually happened**

The background color was purple.

**Additional information**

My initial search of a hex value for indigo led to #4B0082, which appears to be the programming for `Color::INDIGO`. Color pickers seem to show that as more of a purple. However I don't want this issue to get heated over the definition of indigo, so do with this info what you will.

Below is what I see when using `Color::INDIGO`:

![Screenshot from 2021-01-02 12-36-44](https://user-images.githubusercontent.com/8103545/103466159-426ced00-4cf7-11eb-9ea9-46069b24fd06.png)

From a quick Google search, indigo more closely resembles the color below. 
![image](https://user-images.githubusercontent.com/8103545/103466172-75af7c00-4cf7-11eb-92a9-d33abd46fd7c.png)

**Comment by mockersf** [comment]
The colours where added in #859, and should be the HTML colours, so `#4B0082` for indigo (how nice of GitHub to render the colour! Seems to be between the two colours in your issue)
In #616, there was a difference made between rbg and rgb_linear

`Colors::INDIGO` is set up using `rgb_linear` https://github.com/bevyengine/bevy/blob/72b2fc98437f39a8099efcc137697efc32a21800/crates/bevy_render/src/color.rs#L45

If I take the same values and use them as rgb, I get the expected colour:
```rust
  .add_resource(ClearColor(Color::rgb(0.29, 0.0, 0.51)))
```

I'm not very fluent in colour spaces, so would be great if someone that knows those better could take a look at it

**Comment by TheZalli** [comment]
I've read some on color spaces, but to spare you the details (although I do believe every game programmer should understand the basics about [sRGB ](https://en.wikipedia.org/wiki/SRGB) and [gamma correction](https://en.wikipedia.org/wiki/Gamma_correction)), when describing colors you most often want to avoid using the linear colors that haven't been gamma corrected. They are used only when mixing and blending colors, and when sending them to the screen (most often done automatically by the GPU in a shader).

When giving color values, they should always be in sRGB (the **s**tandard RGB space, aka gamma corrected), and this is the colorspace that all programs use when interfacing with users (the values they give, and what you give).

TLDR:

change it from linear to srbg, which is the `Color::rgb` function. It seems that the other colors use linear as well, which is also probably wrong.

**Comment by sburris0** [comment]
I noticed this and did some testing, `rgb_linear()` seems to always produce much lighter colors than expected

**Comment by tigregalis** [comment]
I grabbed the list of 148 (139*) "named colours" in CSS to generate [this code](https://github.com/tigregalis/bevy-html-colours/blob/master/colours.rs) using bevy's own `Color` functions. Do we want this full list?

This is Indigo:
```rust
/// #4B0082 Indigo
pub const INDIGO: Color = Color::rgb_linear(0.07036010921001434326171875, 0., 0.223227977752685546875);
```

```rust
use bevy::prelude::*;

fn main() {
    const INDIGO: Color =
        Color::rgb_linear(0.07036010921001434326171875, 0., 0.223227977752685546875);
    App::build()
        .add_plugins(DefaultPlugins)
        .add_resource(ClearColor(INDIGO))
        .run();
}
```
![image](https://user-images.githubusercontent.com/38416468/104810008-d3ca7c00-582c-11eb-853a-36e117adbea5.png)
![image](https://user-images.githubusercontent.com/38416468/104810031-ff4d6680-582c-11eb-8f8a-9867d126f5c9.png)


*duplicates are:
```rust
"#00FFFF": [
    "Aqua",
    "Cyan",
],
"#FF00FF": [
    "Fuchsia",
    "Magenta",
],
"#778899": [
    "Light Slate Gray",
    "Light Slate Grey",
],
"#708090": [
    "Slate Gray",
    "Slate Grey",
],
"#808080": [
    "Gray",
    "Grey",
],
"#A9A9A9": [
    "Dark Gray",
    "Dark Grey",
],
"#D3D3D3": [
    "Light Gray",
    "Light Grey",
],
"#696969": [
    "Dim Gray",
    "Dim Grey",
],
"#2F4F4F": [
    "Dark Slate Gray",
    "Dark Slate Grey",
],
```

**Comment by mockersf** [comment]
> Do we want this full list?

Yes... and no 😃 

The full fix would be to have a `const` function to create colours with standard rgb hex. To do that, this TODO should be done
https://github.com/bevyengine/bevy/blob/cf0e9f9968bb1bceb92a61cd773478675d35cbd6/crates/bevy_render/src/color.rs#L14

But fixing the colours to have the correct value in linear until this is done is better than what we have now.

**Comment by tigregalis** [comment]
> 
> 
> > Do we want this full list?
> 
> Yes... and no 😃
So the list that's currently in Bevy is a smaller subset of that.
I'll submit a PR later for all of the CSS colours using the generated values. I personally think it's worth having.

> 
> The full fix would be to have a `const` function to create colours with standard rgb hex. To do that, this TODO should be done
> 
> https://github.com/bevyengine/bevy/blob/cf0e9f9968bb1bceb92a61cd773478675d35cbd6/crates/bevy_render/src/color.rs#L14
> 
> But fixing the colours to have the correct value in linear until this is done is better than what we have now.

Maybe I'll have a go at this a bit later on. What should these types be called?

**Comment by mockersf** [comment]
> Maybe I'll have a go at this a bit later on. What should these types be called?

I would go with `Color` for the standard one that everybody is used to, and `LinearColor` for the linear variant, that is the one we have currently
